src/fortran/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
-add_executable(laplace_equation laplace_equation.F90)
+set(ExampleName "laplace_equation")
```

Comment:
We don't need to redefine this variable it has already been defined  in the root directory CMakeLists.txt file.